src/applications/gi/components/profile/AdditionalInformation.jsx

```diff
@@ -1,6 +1,8 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import environment from '../../../../platform/utilities/environment';
```

Comment:
we have a webpack resolve alias for the platform folder imports. relative path is fine but you can also just `import environment from 'platform/utilities/environment'` if you want

Reply:
Ah, I have seen that in the code actually, but got a warning in my IDE so I went with the the relative path. I may update that then merge.